OpenRA.Mods.Common/Widgets/LabelWidget.cs

```diff
@@ -28,15 +30,53 @@ public class LabelWidget : Widget
 		public bool Contrast = ChromeMetrics.Get<bool>("TextContrast");
 		public Color ContrastColor = ChromeMetrics.Get<Color>("TextContrastColor");
 		public bool WordWrap = false;
+
+		[Desc("Space between lines as a percentage (default) of line height or fixed pixel amount.")]
+		public int LineSpacing = 140;
+
+		[Desc("Percentage: line height = LineSpacing% * font size.",
+			"FixedMargin: line height = font size + LineSpacing.",
+			"FixedHeight: line height = LineSpacing.")]
+		public LineSpacingType LineSpacingType = LineSpacingType.Percentage;
+		public LineVAlign LineVAlign = LineVAlign.Middle;
```

Comment:
I'm not convinced we need this.  Top, Middle, Bottom can all be achieved by manipulating the widget position and VAlign, and Collapsed can be achieved by setting the LineSpacing to 100.